Source/UI/NewUI/CommonUI/View/CKOSelectionButton.swift

```diff
@@ -0,0 +1,18 @@
+import UIKit
+
+class CKOSelectionButton: UIView {
```

Comment:
Yes, this will be used in country ticket if this PR is approved

Reply:
Should we then try to manage our merges to include useful stuff? If this is used in a different ticket, it could be declared there. 

From where I'm looking, someone can do that ticket and they may duplicate it with a different name.